django/db/models/fields/__init__.py

```diff
@@ -828,18 +828,17 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
+    @warn_about_renamed_method('Field', '_get_val_from_obj', 'value_from_object',
+                               RemovedInDjango21Warning)
     def _get_val_from_obj(self, obj):
-        if obj is not None:
```

Comment:
ps. Please just reflect on how pathetic you've been and how much of _my time and yours_ you've wasted being cowardly about all this when you could have done a simple merge/reject weeks ago.

A reject still would have made me stop work on composite fields, but at least my time wouldn't have been wasted.